pavelib/utils/test/suites/nose_suite.py

```diff
@@ -157,6 +157,7 @@ def _default_test_id(self):
 
         if self.root == 'lms':
             default_test_id += " {system}/tests.py"
+            default_test_id += " openedx/core/lib/*"
```

Comment:
@awaisdar001 Do these tests also need to be run in the context of Studio also (e.g. with the django settings values and installed applications that we have over on the cms side?)